### PR TITLE
[MIG] base_cron_exclusion for 11.0

### PR DESCRIPTION
**Pull request by bodedra**

### CODE REVIEW

**Comment by max3903** [comment]
@bodedra You are missing the history. There is only one commit and it looks like you did everything...

There should be the commits from the v10 branch + your commit for the migration.

**Comment by pedrobaeza** [comment]
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0

**Comment by max3903** [comment]
@pedrobaeza I just explained @bodedra privately. Proper history should come later today or tomorrow. Thanks!

**Review by max3903 — APPROVED** [review]

**Comment by pedrobaeza** [comment]
You have to clean commit history removing first 2 commits and squashing your 11.0 migration ones

**Comment by max3903** [comment]
@pedrobaeza Done

**Review by pedrobaeza — APPROVED** [review]

File: base_cron_exclusion/models/__init__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
```
Comment (LoisRForgeFlow) [inline base_cron_exclusion/models/__init__.py]: Please do not add copyrights to init files. See template https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py (there are also a discussion on maintainer-tools closed PRs if you are interested)

File: base_cron_exclusion/views/ir_cron_view.xml [hunk 2]
```diff
@@ -1,13 +1,9 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright 2017 Eficent Business and IT Consulting Services S.L.
```
Comment (LoisRForgeFlow) [inline base_cron_exclusion/views/ir_cron_view.xml]: Why you are removing these copyrights?, see template again: https://github.com/OCA/maintainer-tools/blob/master/template/module/views/model_name_view.xml

**Review by LoisRForgeFlow — COMMENTED** [review]
LGTM, just some nitpicking about copyrights that you are changing

File: base_cron_exclusion/models/__init__.py (hunk 1 above)
Comment (bodedra) [reply base_cron_exclusion/models/__init__.py]: Done.

File: base_cron_exclusion/views/ir_cron_view.xml (hunk 2 above)
Comment (bodedra) [reply base_cron_exclusion/views/ir_cron_view.xml]: Done.

**Review by LoisRForgeFlow — APPROVED** [review]
Great, thanks! :+1:

**Review by guewen — APPROVED** [review]

**Pull request merged**